pkg/app/commands.go

```diff
@@ -97,7 +97,7 @@ func defineCommands(
 	computeServe := compute.NewServeCommand(computeCmdRoot.CmdClause, globals, computeBuild, opts.Versioners.Viceroy, data)
 	computeUpdate := compute.NewUpdateCommand(computeCmdRoot.CmdClause, globals, data)
 	computeValidate := compute.NewValidateCommand(computeCmdRoot.CmdClause, globals)
-	configureCmdRoot := configure.NewRootCommand(app, opts.ConfigPath, configure.APIClientFactory(opts.APIClient), globals)
+	configureCmdRoot := configure.NewRootCommand(app, configure.APIClientFactory(opts.APIClient), globals)
```

Comment:
Removed this argument because the value is already exposed as part of the `globals` argument.